test/integration/project-template/conftest.py

```diff
@@ -18,16 +18,18 @@ def new_project(cwd):
     package_name = "package"
 
     subprocess.run(
-        ["cookiecutter", Config.root / "project-template", "-o", cwd, "--no-input",
+        ["cookiecutter", PROJECT_CONFIG.root / "project-template", "-o", cwd, "--no-input",
          f"project_name={project_name}", f"repo_name={repo_name}",
          f"package_name={package_name}",
          ], capture_output=True, check=True)
 
     return cwd / repo_name
 
+
 @pytest.fixture(scope="session", autouse=True)
 def poetry_install(run_command, poetry_path):
     run_command([poetry_path, "install"])
+    run_command([poetry_path, "add", "--group", "dev", PROJECT_CONFIG.root])
```

Comment:
```suggestion
   # Override the dev usage of the python-toolbox to use the unreleased code.
   # This is important so that we can perform needed changes to the project-template
   # in a branch & verify that it works with these tests prior to release.
    run_command([poetry_path, "add", "--group", "dev", PROJECT_CONFIG.root])
```